deps.edn

```diff
@@ -1,18 +1,26 @@
-{:deps {org.clojure/clojure                 {:mvn/version "1.10.0"}
-        org.clojure/clojurescript           {:mvn/version "1.10.773"}
-        reagent/reagent                     {:mvn/version "1.1.0"
-                                             :exclusions
-                                             [cljsjs/react cljsjs/react-dom
-                                              cljsjs/react-dom-server]}
-        re-frame/re-frame                   {:mvn/version "1.2.0"}
-        day8.re-frame/http-fx               {:mvn/version "0.2.3"}
-        lambdaisland/uri                    {:mvn/version "1.4.70"}
-        com.yetanalytics/re-oidc            {:mvn/version "0.0.5"
-                                             :exclusions
-                                             [cljsjs/oidc-client
-                                              reagent/reagent
-                                              re-frame/re-frame]}
-        com.yetanalytics/xapi-schema        {:mvn/version "1.2.1"}}
+{:deps {org.clojure/clojure                     {:mvn/version "1.10.0"}
+        org.clojure/clojurescript               {:mvn/version "1.10.773"}
+        reagent/reagent                         {:mvn/version "1.1.0"
+                                                 :exclusions
+                                                 [cljsjs/react cljsjs/react-dom
+                                                  cljsjs/react-dom-server]}
+        re-frame/re-frame                       {:mvn/version "1.2.0"}
+        day8.re-frame/http-fx                   {:mvn/version "0.2.3"}
+        lambdaisland/uri                        {:mvn/version "1.14.120"}
+        com.yetanalytics/re-oidc                {:mvn/version "0.0.5"
+                                                 :exclusions
+                                                 [cljsjs/oidc-client
+                                                  reagent/reagent
+                                                  re-frame/re-frame]}
+        com.yetanalytics/xapi-schema            {:mvn/version "1.2.1"}
+        com.fasterxml.jackson.core/jackson-core {:mvn/version "2.11.4"}
```

Comment:
Isn't Jackson 2.11.x outdated? The latest non-release-candidate version is 2.14.2.

Reply:
Not trying to go to the latest, just the patched version right now.

Reply:
For minimal disruption.